src/conduits/RwaInputConduit3.sol

```diff
@@ -189,7 +189,7 @@ contract RwaInputConduit3 {
             quitTo = data;
         } else if (what == "to") {
             require(data != address(0), "RwaInputConduit3/invalid-to-address");
-            to = data;
+            to = data; // TB - why is `to` fileable as opposed to RwaInputConduit2?
```

Comment:
Input conduit `to` should be only the `RwaUrn` it's attached to.
In previous versions there not even a possibility to change it.

This address is shared with a crypto broker to allow them send payments back to MakerDAO. We learned that changing addresses is **very** cumbersome because everything is tied up in legal documents, so we added this escape hatch that can only be accessed by governance if we ever need to change it.